lib/render/vp2ShaderFragments/shaderFragments.cpp

```diff
@@ -110,15 +110,19 @@ namespace
 // Fragment registration
 MStatus HdVP2ShaderFragments::registerFragments()
 {
-    MHWRender::MRenderer* theRenderer = MHWRender::MRenderer::theRenderer();
+    // We do not force the renderer to initialize in case we're running in a
+    // headless context. If we cannot get a handle to the renderer or the
+    // fragment manager, we assume that's the case and simply return success.
+    MHWRender::MRenderer* theRenderer =
+        MHWRender::MRenderer::theRenderer(/* initializeRenderer = */ false);
     if (!theRenderer) {
-        return MS::kFailure;
+        return MS::kSuccess;
```

Comment:
These aren't my changes, are the correct?

Reply:
Yes, #130